drag_and_drop_v2/default_data.py

```diff
@@ -97,6 +98,20 @@
       "imageURL": "",
       "id": 3,
     },
+    {
+      "displayName": _("Goes anywhere on the triangle"),
+      "feedback": {
+        "correct": ITEM_ANY_ZONE_FEEDBACK,
+        "incorrect": ITEM_INCORRECT_FEEDBACK
+      },
+      "zones": [
+        TOP_ZONE_ID,
+        BOTTOM_ZONE_ID,
+        MIDDLE_ZONE_ID
+      ],
```

Comment:
@haikuginger Can you update the other items to specify a list of `"zones"` as well?